package io.prizy.domain.contest.exception;

import io.prizy.toolbox.exception.ErrorCode;

@ErrorCode("INVALID_CONTEST_ACCESS_CODE")
public class InvalidContestAccessCodeException extends RuntimeException {
}
